package.json

```diff
@@ -1,27 +1,28 @@
 {
   "name": "chicago-brick",
-  "version": "0.3.7",
+  "version": "0.4.0",
```

Comment:
This is already at head. Is there a way to clean this up?

Reply:
Yep.